### ISSUE TITLE
Connection Timeout

### ISSUE THREAD

**Issue by srjhunjhunwalacorp** [barnstee/UA-CloudCommander#10]
Hello Team,

While using the commander I am getting timeouts, when checked connectivity to OPC UA server seems fine, it sometimes connects fine sometime its shows this random behaviour also with historical and write commands, can you please suggest - 

![Screenshot 2024-06-05 at 3 22 25 PM](https://github.com/OPCFoundation/UA-CloudCommander/assets/64076463/a5bf2d65-0877-44c5-8e4f-88de44620bac)
![Screenshot 2024-06-05 at 3 20 25 PM](https://github.com/OPCFoundation/UA-CloudCommander/assets/64076463/5b218ffd-0f56-482a-8fb4-d557c6560815)

**Comment by srjhunjhunwalacorp** [comment]
I am receiving following logs while starting the container - 

[09:58:02 INF] Log file is: /app/uacloudcommander.logfile.txt
Could not load cert file, creating a new one. This means the new cert needs to be trusted by all OPC UA servers we connect to!
Could not load key file, creating a new one. This means the new cert generated from the key needs to be trusted by all OPC UA servers we connect to!
[09:58:03 INF] Connected to MQTT broker.
[09:58:03 INF] UA Cloud Commander is running.

**Comment by barnstee** [comment]
@srjhunjhunwalacorp , the cert file error is because you are not mapping the PKI directory of the UA Cloud Commander to the Docker host filesystem during Container startup (-v flag in docker). Therefore UA Cloud Commander can't load an existing cert and creates a new one. It seems like you are not using security in your OPC UA server, so this should not be your issue (although not using security is of course an issue!)

It looks like you are using IoT Hub with UA Cloud Commander. Sometimes IoT Hub simply throttles you and you will get timeouts. I would recommend using an Event Hub instead.

**Comment by srjhunjhunwalacorp** [comment]
Yes I am using IoT hub but at the same time I can see that the request is received on the UA cloud commander but it is not initiating a connection towards the OPC UA server as I cant see the connection request on the server side. Earlier it was working fine in the same way but after the new release it started giving timeout issue.

Also I checked the throttling metrics but could not find it happening on the IoT hub.

![image](https://github.com/OPCFoundation/UA-CloudCommander/assets/64076463/9cd3f36b-4c13-4212-bb22-b4077377f034)

**Comment by barnstee** [comment]
Make sure the clock is set correctly in your Docker host.
